core/src/memory.rs

```diff
@@ -909,7 +917,7 @@ mod tests {
             ) {
                 let _ = env_logger::try_init();
 
-                let (max_pages, mem_size, static_pages, allocations) = mem_params;
+                let MemoryParams{max_pages, mem_size, static_pages, allocations} = mem_params;
```

Comment:
I cannot understand is it ok for `fmt` to be without space `{max_pages`?

Reply:
I guess it's ok, if I understood you correctly.